sigma.css

```diff
@@ -184,6 +184,7 @@ textarea {
 div#container-wrap {
 	background: url('https://scpwiki.github.io/sigma/images/body_bg.svg') top
 		left repeat-x;
+	background-size: 100px 400px;
```

Comment:
This should be fine also. It feels redundant since modern browsers, again, just do this automatically. Still, it should not do any harm.